cleanlab/regression/rank.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+from cleanlab.outlier import OutOfDistribution
+from sklearn.neighbors import NearestNeighbors
+from cleanlab.internal.regression_utils import assert_valid_inputs
+from typing import Dict, Callable
+
+""" Generates label quality scores for every sample in regression dataset """
+
+
+def get_label_quality_scores(
+    labels: np.ndarray,
+    predictions: np.ndarray,
```

Comment:
let's declare these as array_like?
C.f. https://github.com/cleanlab/cleanlab/blob/00781ce984e78a4b17cefd28f9d49c8218ac717f/cleanlab/outlier.py#L195

That covers anything you can call `np.asarray(labels)` to get back a 1D np.ndarray for, like a list of numbers